projects/framework-houdini/source/ftrack_framework_houdini/__init__.py

```diff
@@ -0,0 +1,157 @@
+
+    __version__ = get_version(
+        os.path.basename(os.path.dirname(__file__)),
+        os.path.dirname(os.path.dirname(os.path.dirname(__file__))),
+    )
+except Exception:
+    __version__ = '0.0.0'
+
+configure_logging(
+    'ftrack_framework_houdini',
+    extra_modules=['ftrack_qt'],
+    propagate=False,
+)
+
+logger = logging.getLogger(__name__)
+logger.debug('v{}'.format(__version__))
+
+
+@run_in_main_thread
+def on_run_dialog_callback(dialog_name, tool_config_names, docked):
+    client_instance.run_dialog(
+        dialog_name,
+        dialog_options={
+            'tool_config_names': tool_config_names,
+            'docked': docked,
+        },
+        dock_func=dock_houdini_right,
+    )
+
+
+def get_ftrack_menu():
+    '''Construct the xml representation of the ftrack menu.'''
+    root = ET.Element("mainMenu")
+    menubar = ET.SubElement(root, "menuBar")
+    ftrack_menu = ET.SubElement(menubar, "subMenu")
+    label = ET.SubElement(ftrack_menu, "label")
+    label.text = "ftrack"
+    insert_before = ET.SubElement(ftrack_menu, "insertBefore")
+    insert_before.text = "help_menu"
+    return (root, ftrack_menu)
+
+
+def bootstrap_integration(framework_extensions_path):
+    '''
+    Initialise Houdini Framework integration
+    '''
+    logger.debug(
+        'Houdini standalone integration initialising, extensions path:'
+        f' {framework_extensions_path}'
+    )
+
+    global client_instance
+
+    # Create ftrack session and instantiate event manager
+    session = ftrack_api.Session(auto_connect_event_hub=False)
+    event_manager = EventManager(
+        session=session, mode=constants.event.LOCAL_EVENT_MODE
+    )
+    logger.debug(f"framework_extensions_path:{framework_extensions_path}")
+    # Instantiate registry
+    registry_instance = Registry()
+    registry_instance.scan_extensions(paths=framework_extensions_path)
+
+    # Instantiate Host and Client
+    Host(event_manager, registry=registry_instance)
+    client_instance = Client(event_manager, registry=registry_instance)
+
+    # Init tools
+    dcc_config = registry_instance.get_one(
+        name='framework-houdini', extension_type='dcc_config'
+    )['extension']
+
+    logger.debug(f'Read DCC config: {dcc_config}')
+
+    # Generate ftrack menu
+    root, ftrack_menu = get_ftrack_menu()
+
+    for tool in dcc_config['tools']:
+        menu_item = ET.SubElement(ftrack_menu, "scriptItem")
+        menu_item.set("id", tool['name'])
+        label = ET.SubElement(menu_item, "label")
+        label.text = tool['label']
+        menu_item_script = ET.SubElement(menu_item, "scriptCode")
+        menu_item_script.text = _get_menu_item_script(tool)
+
+    # Convert xml to string
+    # Unescaping and decoding to avoid ending up with encoded CDATA
+    xml = unescape(ET.tostring(root).decode())
+
+    # Find the temp file where to write xml for MainMenuCommon.xml
+    xml_menu_file_folder = os.environ['FTRACK_HOUDINI_XML_MENU_FILE']
+
```

Comment:
@seebergs  @henriknorin-ftrack 

bootstrap seems to be missing of usagetracker as[ other integrations](https://github.com/ftrackhq/integrations/blob/main/projects/framework-maya/source/ftrack_framework_maya/__init__.py#L157)

Reply:
also reported during the integration initialization as: 

`WARNING - MainThread - 2024-05-28 10:36:38,185 - ftrack_utils:usage - UsageTracker instance is not set. Try to initialize a default one.`

Reply:
Correct @hdd , please refer to: https://github.com/ftrackhq/integrations/blob/f673fdf70776a1c0c8b2f1925158224dc9795160/projects/framework-maya/source/ftrack_framework_maya/__init__.py#L156